src/components/bookList.js

```diff
@@ -1,13 +1,22 @@
 /* eslint-disable */
 
 import React from 'react';
-import { useStore } from 'react-redux';
+import { useStore, useDispatch } from 'react-redux';
 import Book from './book';
+import * as Actions from '../actions/index';
 
 const bookList = () => {
   // const books = useSelector(state => state.books);
```

Comment:
## Remove commented out code
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))